examples/css3d_interactive_raycast.html

```diff
@@ -0,0 +1,164 @@
+onRenderFcts.push(function(){
+        controls.update();
+})
+
+//////////////////////////////////////////////////////////////////////////////////
+//		initialize renderer for css coverflow
+//////////////////////////////////////////////////////////////////////////////////
+
+// var rendererCss = new THREE.CSS3DStereoRenderer();
+var rendererCss = new THREE.CSS3DRenderer();
+
+rendererCss.separation = 2;
+rendererCss.targetDistance = 500;
+
+rendererCss.setSize( window.innerWidth, window.innerHeight );
+rendererCss.domElement.style.position = 'absolute';
+rendererCss.domElement.style.top = '0px';
+rendererCss.domElement.style.pointerEvents = 'none';
+rendererCss.domElement.classList.add('css3dContainer')
+document.body.appendChild( rendererCss.domElement );
+// render the scene
+onRenderFcts.push(function(){
+        // to actually render the scene   
+        rendererCss.render( sceneCss, camera );
+})
+
+
+var sceneCss = new THREE.Scene()
+
+
+var domElement = document.createElement('div');
+domElement.style.width = '5px'
+domElement.style.height = '1px'
+domElement.style.backgroundColor = 'rgba(255,0,0, 0.5)'
+
+var css3DObject = new THREE.CSS3DObject( domElement );          
+sceneCss.add(css3DObject);
+
+//////////////////////////////////////////////////////////////////////////////
+//              Code Separator
+//////////////////////////////////////////////////////////////////////////////
+
+var light	= new THREE.AmbientLight( 0x080808 )
+scene.add( light )
+// add a light in front
+var light2	= new THREE.DirectionalLight('white', 0.5)
+light2.position.set(0.5, 2, 0.1)
+scene.add( light2 )
+
+//////////////////////////////////////////////////////////////////////////////
+//              Code Separator
+//////////////////////////////////////////////////////////////////////////////
+
+var geometry = new THREE.BoxGeometry( 5, 1, 0.1 );
+var material = new THREE.MeshNormalMaterial();
+var cube = new THREE.Mesh( geometry, material );
+cube.position.z=3
+scene.add( cube );
+
+// grid
+var gridHelper = new THREE.GridHelper( 10, 1 ); 
+scene.add( gridHelper );
+
+//////////////////////////////////////////////////////////////////////////////
+//              STEREO rendererCss
+//////////////////////////////////////////////////////////////////////////////
+
+onRenderFcts.push(function(){
+        renderer.render( scene, camera );
+})
+
+//////////////////////////////////////////////////////////////////////////////
+//              Code Separator
+//////////////////////////////////////////////////////////////////////////////
+
+var mouse = new THREE.Vector2();
+
+document.addEventListener( 'mousemove', onDocumentMouseMove, false )
+function onDocumentMouseMove( event ) {
+	event.preventDefault();
+	mouse.x = ( event.clientX / window.innerWidth ) * 2 - 1;
+	mouse.y = - ( event.clientY / window.innerHeight ) * 2 + 1;
+        
+        computeIntersect(mouse)
+}
+
+function computeIntersect(mouse){
+        var raycaster = new THREE.Raycaster();
```

Comment:
thanks for reviewing it! i did the change you asked and remove some other debug stuff. i hope it is better now.